Refresh.GameServer/Types/Levels/Categories/ContestCategory.cs

```diff
@@ -0,0 +1,30 @@
+using Bunkum.Core;
+using Refresh.GameServer.Database;
+using Refresh.GameServer.Endpoints.Game.Levels.FilterSettings;
+using Refresh.GameServer.Services;
+using Refresh.GameServer.Types.Contests;
+using Refresh.GameServer.Types.UserData;
+
+namespace Refresh.GameServer.Types.Levels.Categories;
+
+public class ContestCategory : LevelCategory
+{
+    public ContestCategory() : base("contest", Array.Empty<string>(), false)
+    {
+        this.Name = "Contests";
+        this.Description = "Levels from a contest.";
+        this.FontAwesomeIcon = "certificate";
+        this.IconHash = "g820608";
+        this.Hidden = true;
+    }
+    
+    public override DatabaseList<GameLevel>? Fetch(RequestContext context, int skip, int count, MatchService matchService,
+        GameDatabaseContext database, GameUser? user, LevelFilterSettings levelFilterSettings)
+    {
+        string? contestId = context.QueryString["contest"];
+        GameContest? contest = database.GetContestById(contestId);
+        if (contest == null) return null;
+        
+        return database.GetLevelsFromContest(contest, count, skip, user, levelFilterSettings);
+    }
```

Comment:
imo this should default to the latest contest (returning null when there is no active contest), so and also be unhidden and added to ingame categories, so for LBP3 where we can do custom level categories, the current latest contest will display as a level category.

Also did you forget to add this category to [here](https://github.com/LittleBigRefresh/Refresh/blob/4035585e12ca8f0190787c99dc16f57f08c6f424/Refresh.GameServer/Types/Levels/Categories/CategoryService.cs#L12)?

Reply:
I did forget, yes, thank you.

Good points, though, I'll change things to match that. I made the category pretty early on and then forgot about it so I haven't considered much about it.